Applications/utils/jni/meson.build

```diff
@@ -0,0 +1,24 @@
+utils_inc = include_directories('./includes')
+
+nntrainer_utils_source = [
+  'bitmap_helpers.cpp'
+]
+
+shared_library('nntrainer_utils',
+	nntrainer_utils_source,
+	include_directories: utils_inc,
+	install: get_option('install-app'),
+	install_dir: application_install_dir
+)
+
+nntrainer_utils_lib = static_library('nntrainer_utils',
```

Comment:
Could we rename this as nntrainer_app_utils or equivalent?
nntrainer_utils sounds like it is part of the library (if it is planned to be, I think it's fine)

Reply:
fixed. thanks